gms/inet_address.hh

```diff
@@ -69,9 +69,9 @@ public:
         using namespace std::rel_ops;
         return x._addr != y._addr;
     }
-    friend inline bool operator<(const inet_address& x, const inet_address& y) noexcept {
-        return x.bytes() < y.bytes();
-    }
+    friend inline auto operator<=>(const inet_address& x, const inet_address& y) noexcept {
```

Comment:
no need to mark this function friend. as `bytes()` is a public function.

Reply:
true, I just preserved the existing declaration style